### PR TITLE
Add reactions and edits with PR cherrypick

### PR DESCRIPTION
**Pull request by estellecomment**
https://github.com/tchapgouv/tchap-web/issues/166

### CODE REVIEW

**Comment by estellecomment** [comment]
Work done : added PRs in the order they were added in element repo.

PR 2935 : message action bar
commit 0bc54ba9043b933892eb55d72799babfc821a502, parents 3c880b4d75, 739c8c0314

(Could also add maybe : Commit : update action bar colors for dark theme : 4b50d2a2bfa6b4d6fbc8c40fae5d370c1eb0c817)

PR : https://github.com/matrix-org/matrix-react-sdk/pull/2937
`git cherry-pick -m 1  a74824aef48a0c19495b224a55e938e7a8771391`

PR : https://github.com/matrix-org/matrix-react-sdk/pull/2940
`git cherry-pick -m 1 6fcc418acce829c0c65fec5bd0eeeb3d578b44da`

Result : buttons don't show, and reactions either. Need to understand better what's going on.

File: src/components/views/rooms/EventTile.js [hunk 1]
```diff
@@ -761,6 +745,8 @@ module.exports = withMatrixClient(React.createClass({
                                            onHeightChanged={this.props.onHeightChanged} />
                             { keyRequestInfo }
                             { editButton }
```
Comment (jdauphant) [inline src/components/views/rooms/EventTile.js:747]: le bouton n'existe pas

**Comment by jdauphant** [comment]
I took a look at the history of the ReactionRow and I think we need https://github.com/matrix-org/matrix-react-sdk/pull/2954/files for data

**Comment by estellecomment** [comment]
You need to enable reactions in config.json in tchap-web : 
```
    "features": {
        "feature_reactions": "enable"
    }
```

**Comment by estellecomment** [comment]
This breaks because our version of matrix-js-sdk is too old. Our version is from 2019-04-16.

We need function `getRelationsForEvent` in src/models/event-timeline-set.js .
Added in commit https://github.com/matrix-org/matrix-js-sdk/commit/6078bbbe2475a1b95d211b64caa588b375c371d9 on May 8, 2019.

**Comment by jdauphant** [comment]
Add https://github.com/matrix-org/matrix-react-sdk/pull/3005 to fix a bug

**Comment by jdauphant** [comment]
We are moving to #370

**Pull request closed without merging**